File: engine/b/src/main/java/net/crazyinvestor/engine_b/dto/BithumbWSTransactionResponseContent.java
package net.crazyinvestor.engine_b.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public final class BithumbWSTransactionResponseContent {
    private String symbol;
    private String buySellGb;
    private String contPrice;
    private String contQty;
    private String contAmt;
    private String contDtm;
    private String updn;
}

File: engine/a/ticker_producer_engine/src/main/kotlin/net/crazyinvestor/ticker_producer_engine/dto/bithumb/response/BithumbWSTransactionResponseContent.kt
package net.crazyinvestor.ticker_producer_engine.dto.bithumb.response

class BithumbWSTransactionResponseContent (
    val symbol: String?,
    val buySellGb: String?,
    val contPrice: String?,
    val contQty: String?,
    val contAmt: String?,
    val contDtm: String?,
    val updn: String?
){
    constructor(): this(null,null,null,null,null,null,null){}
}